modules/interop/zio2/logging/src/test/scala-2/tofu/logging/zlogs/ZLogsSpec.scala

```diff
@@ -0,0 +1,114 @@
+package tofu.logging.zlogs
+
+import ch.qos.logback.classic.Level
+import io.circe.{Json, JsonObject}
+import tofu.logging.impl.ContextMarker
+import tofu.logging.zlogs.TestStuff._
+import tofu.logging.zlogs.TofuDefaultContextSpec.{testCount, testStatus, testUser}
+import tofu.logging.{LogTree, LoggedValue}
+import zio._
+import zio.test._
+
+object ZLogsSpec extends ZIOSpecDefault {
+
+  val loggerName: String = this.getClass.getName.replace("$", "")
```

Comment:
`case object ZLogsSpec` & `this.productPrefix`

Reply:
this should be with package prefix